.drone.jsonnet

```diff
@@ -214,7 +214,7 @@ local pull_image(image, volumes=[]) = {
 # r_path          path to R executable
 # expr            expression to run
 local run_r_expression(r_path, expr) =
-  std.join(" ", [r_path, "-e", std.escapeStringBash(expr)]);
+  std.join(" ", [r_path, "--no-init-file", "-e", std.escapeStringBash(expr)]);
```

Comment:
this is so drone _doesn't_ use renv right? so that the containers are still controlling the packages?

Reply:
this is not about drone "not using renv" per say - its about R not sourcing the .Rprofile when it starts up, therefore renv does not get activated

Reply:
Exactly. We want the package set to be dictated by the container, not renv. renv is activated in `.Rprofile`, but because the renv library is not committed to git, renv won't find any of the package dependencies (which we don't want to use in any case), so `check()` will fail. So as @dpastoor says, we bypass activating renv by skipping `.Rprofile`.